src/common-ssh/common-ssh/user.h

```diff
@@ -104,5 +110,22 @@ void guac_common_ssh_user_set_password(guac_common_ssh_user* user,
 int guac_common_ssh_user_import_key(guac_common_ssh_user* user,
         char* private_key, char* passphrase);
 
+/**
+ * Imports the given public key, associating that key with the given user.
+ * If the public key is imported successfully, it will be used for
+ * future authentication attempts.
+ *
+ * @param user
+ *     The user to associate with the given private key.
+ *
+ * @param public_key
```

Comment:
This looks to be missing the documentation on the parameter.

Reply:
Added missing documentation